File: app/src/main/java/com/example/bmicalculator/Pytanie.java
package com.example.bmicalculator;

public class Pytanie {

    String pytanie;
    String[] odpowiedzi = new String[4];
    String dobraOdp;

    Pytanie(String pytanie, String[] odpowiedzi, String dobraOdp) {

        this.pytanie = pytanie;
        this.odpowiedzi = odpowiedzi;
        this.dobraOdp = dobraOdp;

    }
}

File: app/src/main/java/com/example/bmicalculator/kotlinclasses/Pytanie.kt
package com.example.bmicalculator.kotlinclasses

class Pytanie internal constructor(var pytanie: String, odpowiedzi: Array<String?>, dobraOdp: String) {

    var odpowiedzi = arrayOfNulls<String>(4)
    var dobraOdp: String

    init {

        this.odpowiedzi = odpowiedzi
        this.dobraOdp = dobraOdp
    }
}
